src/main/scala/huancun/HuanCun.scala

```diff
@@ -244,8 +244,14 @@ class HuanCun(implicit p: Parameters) extends LazyModule with HasHuanCunParamete
     val io = IO(new Bundle {
       val perfEvents = Vec(banks, Vec(numPCntHc,Output(UInt(6.W))))
       val ecc_error = Valid(UInt(64.W))
+      val debugTopDown = new Bundle {
+        val robHeadPaddr = Vec(cacheParams.hartIds.length, Flipped(Valid(UInt(36.W))))
+        val addrMatch = Vec(cacheParams.hartIds.length, Output(Bool()))
+      }
     })
 
+    io.debugTopDown <> DontCare
```

Comment:
Is 'DontCare' here necessary?
We prefer to eliminating DontCare forever :)

Reply:
I see, let me change it.